samples/bluetooth/mesh/dfu/distributor/sysbuild/mcuboot/socs/nrf54l10_cpuapp.conf

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+################################################################################
+CONFIG_BOOT_MAX_IMG_SECTORS=256
+
+# Ensure that the SPI NOR driver is disabled by default
+CONFIG_SPI_NOR=n
```

Comment:
Hm... This disables SPI_NOR for mcuboot but this: https://github.com/nrfconnect/sdk-nrf/pull/24880/files#diff-eadce601741303ceaec217201d77380136083f763b38570f3417112156c06e12R3 enables. Seems like controversal to each other settings. Could elaborate why is it required?